studio/components/layouts/ProjectLayout/ProjectPausedState.tsx

```diff
@@ -95,56 +98,64 @@ const ProjectPausedState = ({ product }: ProjectPausedStateProps) => {
                     'Restore this project and get back to building the next big thing!'
                   )}
                 </p>
-                {isFreePlan && (
-                  <p className="text-sm text-scale-1100 text-center">
-                    You can also prevent project pausing in the future by upgrading to Pro.
-                  </p>
-                )}
-                {!isFreePlan && (
+                {!haveOverdueInvoices ? (
+                  isFreePlan ? (
+                    <p className="text-sm text-scale-1100 text-center">
+                      You can also prevent project pausing in the future by upgrading to Pro.
+                    </p>
+                  ) : (
+                    <p className="text-sm text-scale-1100 text-center">
+                      Unpaused projects count towards compute usage. For every hour your instance is
+                      active, we'll bill you based on the instance size of your project. See{' '}
+                      <Link href="https://www.notion.so/supabase/Organization-Level-Billing-707638e35c92489995dc3ac991a324d1">
+                        <a target="_blank" rel="noreferrer" className="underline">
+                          Compute Instance Usage Billing
+                        </a>
+                      </Link>{' '}
+                      for more details.
+                    </p>
+                  )
+                ) : (
                   <p className="text-sm text-scale-1100 text-center">
-                    Unpaused projects count towards compute usage. For every hour your instance is
-                    active, we'll bill you based on the instance size of your project. See{' '}
-                    <Link href="https://www.notion.so/supabase/Organization-Level-Billing-707638e35c92489995dc3ac991a324d1">
-                      <a target="_blank" rel="noreferrer" className="underline">
-                        Compute Instance Usage Billing
-                      </a>
-                    </Link>{' '}
-                    for more details.
+                    You have outstanding invoices that need to be paid before you’re able to resume
+                    this project. Please contact support if you need more help.
```

Comment:
Should have a CTA to get to the outstanding invoices (/org/<slug>/invoices)

contact support should also be a link to to the support/new form